src/app/routes/getInitialData/utils/preprocessorRulesConfig/index.js

```diff
@@ -0,0 +1,37 @@
+import applyTimestampRules from '#lib/utilities/preprocessor/rules/timestamp';
+import addIdsToBlocks from '#lib/utilities/preprocessor/rules/addIdsToBlocks';
+import radioPageAddIdsToBlocks from '#lib/utilities/preprocessor/rules/radioPage/addIdsToBlocks';
```

Comment:
Having 2 different `addIdsToBlocks` is annoying. It's currently needed because radioPage gets `content.blocks` and other pages get `content.model.blocks`, probably something to refactor after this PR IMO